tutorials/releases.md

```diff
@@ -0,0 +1,311 @@
+apt-key add - < Release.key
+apt-get update
+apt-get install cri-o
+```
+### Fedora 31 or later
+```
+sudo dnf module enable cri-o:1.18
+sudo dnf install cri-o
+```
+### Ubuntu 20.04
+For Ubuntu 20.04 run the following:
+Keep in mind that the owner of the key may distribute updates, packages and repositories that your system will trust.
+```
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.18:/1.18.0/xUbuntu_20.04/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0.list"
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0/xUbuntu_20.04/Release.key -O Release.key
+sudo apt-key add - < Release.key
+sudo apt-get update
+sudo apt-get install cri-o
+```
+# 1.17
+## 1.17.4
+### CentOS 7
+For CentOS 7 run the following as root:
+```
+cd /etc/yum.repos.d/
+wget https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4/CentOS_7/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4.repo
+yum install cri-o
+```
+### CentOS 8
+For CentOS 8 run the following as root:
+```
+cd /etc/yum.repos.d/
+wget https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4/CentOS_8/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4.repo
+yum install cri-o
+```
+### CentOS 8 Stream
+For CentOS 8 Stream run the following as root:
+```
+cd /etc/yum.repos.d/
+wget https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4/CentOS_8_Stream/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4.repo
+yum install cri-o
+```
+### Debian Unstable
+For Debian Unstable run the following as root:
+Keep in mind that the owner of the key may distribute updates, packages and repositories that your system will trust.
+```
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.17:/1.17.4/Debian_Unstable/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4/Debian_Unstable/Release.key -O Release.key
+apt-key add - < Release.key
+apt-get update
+apt-get install cri-o
+```
+### Debian Testing
+For Debian Testing run the following as root:
+Keep in mind that the owner of the key may distribute updates, packages and repositories that your system will trust.
+```
+echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.17:/1.17.4/Debian_Testing/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4.list
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.4/Debian_Testing/Release.key -O Release.key
+apt-key add - < Release.key
+apt-get update
+apt-get install cri-o
```

Comment:
If you're mentioning testing repo links in docs, then risk-taking is best done in the Unstable project (sibling project of stable and testing). Don't make any huge changes in testing unless you've first tested in unstable.